static/js/tools/shared/vis_tools/landing_page_example_links.ts

```diff
@@ -0,0 +1,129 @@
+      title: intl.formatMessage({
+        id: "attainment_of_bachelor_degree_in_colorado",
+        defaultMessage: "Attainment of bachelor's degree in Colorado",
+        description:
+          "Title of a map plotting the statistical variable 'bachelors degree attainment' for counties of Colorado",
+      }),
+      url: "/tools/map#%26sv%3DCount_Person_EducationalAttainmentBachelorsDegreeOrHigher%26pc%3D1%26denom%3DCount_Person%26pd%3DgeoId%2F08%26ept%3DCounty",
+    },
+  ],
+  scatterLinks: [
+    {
+      id: "scatter_prevalence_of_coronary_heart_disease_vs_projected_temperature_rise_in_the_usa",
+      title: intl.formatMessage({
+        id: "prevalence_of_coronary_heart_disease_vs_projected_temperature_rise_in_the_usa",
+        defaultMessage:
+          "Prevalence of coronary heart disease vs. projected temperature rise in the USA",
+        description:
+          "Title of a scatter plot showing the correlation between coronary heart disease and projected temperature rise in the USA",
+      }),
+      url: "/tools/scatter#%26svx%3DDifferenceRelativeToBaseDate2006_Max_Temperature_RCP45%26dx%3DCount_Person%26svy%3DPercent_Person_WithCoronaryHeartDisease%26dy%3DCount_Person%26epd%3Dcountry%2FUSA%26ept%3DCounty%26qd%3D1%26dd%3D1%26pp%3D",
+    },
+    {
+      id: "scatter_literate_population_per_capita_vs_population_below_poverty_level_per_capita_for_states_in_india",
+      title: intl.formatMessage({
+        id: "literate_population_per_capita_vs_population_below_poverty_level_per_capita_for_states_in_india",
+        defaultMessage:
+          "Literate population per capita vs. population below poverty level per capita for states in India",
+        description:
+          "Title of a scatter plot showing the correlation between the literate population per capita and population below poverty level per capita for states in India",
+      }),
+      url: "/tools/scatter#%26svx%3DCount_Person_BelowPovertyLevelInThePast12Months_AsFractionOf_Count_Person%26pcx%3D1%26dx%3DCount_Person%26svy%3DCount_Person_Literate%26pcy%3D1%26dy%3DCount_Person%26epd%3Dcountry%2FIND%26ept%3DAdministrativeArea1%26ct%3D1%26pp%3D",
+    },
+  ],
+  timelineLinks: [
+    {
+      id: "timeline_water_withdrawal_rate_in_california",
+      title: intl.formatMessage({
+        id: "water_withdrawal_rate_in_california",
+        defaultMessage: "Water withdrawal rate in California",
+        description:
+          "Title of a line chart plotting the statistical variable 'water withdrawal rate' in California",
+      }),
+      url: "/tools/timeline#place=geoId%2F06&statsVar=WithdrawalRate_Water_Thermoelectric__WithdrawalRate_Water_PublicSupply__WithdrawalRate_Water_Irrigation__WithdrawalRate_Water_Aquaculture",
+    },
+    {
+      id: "timeline_university_towns_by_income",
+      title: intl.formatMessage({
+        id: "university_towns_by_income",
+        defaultMessage: "University towns by income",
```

Comment:
@miss-o-soup In the mocks, you wrote "University towns by income _in Cambridge_", but I assume you meant to use the current example we have on prod, which shows a chart for multiple university towns, not just Cambridge. Let me know if you wanted income in Cambridge explicitly, and I'll change the text and url.